helpers/management.py

```diff
@@ -0,0 +1,89 @@
+
+import newrelic.agent
+from discord import Member, ChannelType, Permissions
+
+
+add_message_file = 'addMessage.shelve'
```

Comment:
just had a search through the code not entirely sure what `add_message_file = 'addMessage.shelve'` is, it's not used lol
also having a separate file just for one variable can quite quickly make a project messy
also why `__init__.py`?  what's it initalising? having it in the current file does the job

Reply:
oh does your code not have a customisable add message depending on the server? these were stored in a shelve file, in which case this could just be removed. 

__init__.py are python package initialisers, https://careerkarma.com/blog/what-is-init-py/  I like putting any fixed variables in them so they can be used across the package and makes them easier to find. for later configuring.

Reply:
I removed that pile of code with the intention of rebuilding it
then decided that it wasn't a high priority feature